pkg/stanza/operator/helper/operator.go

```diff
@@ -75,7 +75,7 @@ func (c BasicConfig) Build(set component.TelemetrySettings) (BasicOperator, erro
 type BasicOperator struct {
 	OperatorID   string
 	OperatorType string
-	*zap.SugaredLogger
+	logger       *zap.Logger
```

Comment:
Same as above, why not store `component.TelemetrySettings` here?

Reply:
Done